modules/recording/recordingXMLUtils.ts

```diff
@@ -2,17 +2,31 @@
  * A collection of utility functions for taking in XML and parsing it to return
  * certain values.
  */
-export default {
+
+export interface IFocusRecordingUpdate {
+    error: string | null;
+    initiator: string | null;
+    recordingMode: string | null;
+    sessionID: string | null;
+    status: string | null;
+}
+
+export interface IHiddenDomainUpdate {
+    liveStreamViewURL: string | null | undefined;
+    mode: string | null | undefined;
+    sessionID: string | null | undefined;
+}
+
+const recordingXMLUtils = {
```

Comment:
Rather than exporting an object with a few stateless methods, just exports each method independencty and adjust the import please.